build-gluster-org/scripts/centos-regression.sh

```diff
@@ -117,17 +108,9 @@ else
 fi
 
 RET=$?
-if [ $RET = 0 ]; then
-    V="+1"
-    VERDICT="SUCCESS"
-else
-    V="-1"
-    VERDICT="FAILED"
-fi
+
 echo "Logs are archived at Build artifacts: https://build.gluster.org/job/${JOB_NAME}/${UNIQUE_ID}"
 
-# Update Gerrit with the success/failure status
 sudo mv /tmp/gluster_regression.txt $WORKSPACE || true
 sudo chown jenkins:jenkins gluster_regression.txt || true
-vote_gerrit "$V" "$VERDICT" gluster_regression.txt
```

Comment:
wouldn't it be easier to just replace vote_gerrit by a log intead of a vote ?

Reply:
Good option. Can do it.